basic-javascript/tests/project-1.test.js

```diff
@@ -10,178 +11,256 @@ const funcs = require('../src/project-1');
+      expect(roundUp(3.1)).to.equal(4);
     });
   });
 
   describe('`addExclamationPoint`', () => {
+    const addExclamationPoint = funcs.addExclamationPoint;
     it('should be a function', () => {
-      const addExclamationPoint = funcs.addExclamationPoint;
-      assert.typeOf(addExclamationPoint, 'string');
+      assert.typeOf(addExclamationPoint, 'function');
+    });
+    it('should add an exclamation point to end of string', () => {
+      expect(addExclamationPoint('Hurray')).to.equal('Hurray!');
     });
   });
 
   describe('`combineNames`', () => {
+    const combineNames = funcs.combineNames;
     it('should be a function', () => {
-      const combineNames = funcs.combineNames;
-      assert.typeOf(combineNames, 'string');
+      assert.typeOf(combineNames, 'function');
+    });
+    it('should combine a first and last name', () => {
+      expect(combineNames('Emma', 'BoBemma')).to.equal('Emma BoBemma');
     });
   });
 
   describe('`getGreeting`', () => {
+    const getGreeting = funcs.getGreeting;
     it('should be a function', () => {
-      const getGreeting = funcs.getGreeting;
-      assert.typeOf(getGreeting, 'string');
+      assert.typeOf(getGreeting, 'function');
+    });
+    it('should return a personalized greeting', () => {
+      expect(getGreeting('Ivan')).to.equal('Hello Ivan!');
     });
   });
 
   describe('`getRectangleArea`', () => {
+    const getRectangleArea = funcs.getRectangleArea;
     it('should be a function', () => {
-      const getRectangleArea = funcs.getRectangleArea;
-      assert.typeOf(getRectangleArea, 'string');
+      assert.typeOf(getRectangleArea, 'function');
+    });
+    it('should return the area of a rectangle', () => {
+      expect(getRectangleArea(2,3)).to.equal(6);
     });
   });
 
   describe('`getTriangleArea`', () => {
+    const getTriangleArea = funcs.getTriangleArea;
     it('should be a function', () => {
-      const getTriangleArea = funcs.getTriangleArea;
-      assert.typeOf(getTriangleArea, 'string');
+      assert.typeOf(getTriangleArea, 'function');
+    });
+    it('should return the area of a triangle', () => {
+      expect(getTriangleArea(3,4)).to.equal(6);
     });
   });
 
   describe('`getCircleArea`', () => {
+    const getCircleArea = funcs.getCircleArea;
     it('should be a function', () => {
-      const getCircleArea = funcs.getCircleArea;
-      assert.typeOf(getCircleArea, 'string');
+      assert.typeOf(getCircleArea, 'function');
+    });
+    it('should return the area of a circle', () => {
+      expect(getCircleArea(4)).to.equal(Math.PI*4*4);
     });
   });
 
   describe('`getRectangularPrismVolume`', () => {
+    const getRectangularPrismVolume = funcs.getRectangularPrismVolume;
     it('should be a function', () => {
-      const getRectangularPrismVolume = funcs.getRectangularPrismVolume;
-      assert.typeOf(getRectangularPrismVolume, 'string');
+      assert.typeOf(getRectangularPrismVolume, 'function');
     });
-  });
```

Comment:
Missing a closing set of brackets here `})` which prevented the test from running.